collections/ansible_collections/rhsiqe/skupper/roles/install_skupper/tasks/main.yml

```diff
@@ -15,25 +15,60 @@
     version: "{{ install_skupper_version }}"
     update: true
 
-- name: Install Skupper using Helm
+- name: Generating HELM files
+  ansible.builtin.command:
+    cmd: make generate-skupper-helm-chart
+    chdir: "{{ skupper_folder }}"
+  register: generate_helm_files
+
+- name: Print the output of the command
+  ansible.builtin.debug:
+    msg: "{{ generate_helm_files.stdout_lines }}"
+
+- name: Install Skupper using Helm Namespace Scope
   kubernetes.core.helm:
-    chart_ref: "{{ skupper_folder }}/charts/skupper-setup"
+    chart_ref: "{{ skupper_folder }}/charts/skupper"
     name: "{{ install_skupper_skupper_release_name }}"
-    namespace: "{{ install_skupper_namespace | default('default') }}"
+    kubeconfig: "{{ kubeconfig }}"
+    values:
+      scope: "{{ install_skupper_scope }}"
     wait: true
     replace: false
+    namespace: "{{ install_skupper_namespace }}"
+  when: install_skupper_scope  == 'namespace'
+  ignore_errors: true
+
+- name: Create skupper-operator namespace based on the value of install_skupper_operator_namespace
+  kubernetes.core.k8s:
+    definition:
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: "{{ install_skupper_namespace }}"
+    kubeconfig: "{{ kubeconfig }}"
+  when: install_skupper_scope == 'cluster'
+
+- name: Install Skupper using Helm (Cluster Scope)
+  kubernetes.core.helm:
+    chart_ref: "{{ skupper_folder }}/charts/skupper"
+    name: "{{ install_skupper_skupper_release_name }}"
     kubeconfig: "{{ kubeconfig }}"
     values:
       scope: "{{ install_skupper_scope }}"
+    wait: true
+    replace: false
+    release_namespace: "{{ install_skupper_namespace }}"
+  when: install_skupper_scope == 'cluster'
   ignore_errors: true
 
 - name: Validate Skupper controller pod status
   kubernetes.core.k8s_info:
     api_version: v1
     kind: Pod
-    namespace: skupper
     namespace: "{{ install_skupper_namespace | default('skupper') }}"
     kubeconfig: "{{ kubeconfig }}"
+    label_selectors:
+      - app.kubernetes.io/part-of=skupper
```

Comment:
will it also pick the skupper-network-observer pods ?

What about this : 

   label_selectors:
      - app.kubernetes.io/part-of in (skupper, skupper-network-observer)

Reply:
Addressed at: 9127d4e

Closes #9